### ISSUE TITLE
Fork this into JS module

### ISSUE THREAD

**Issue by madeleineostoja** [SimplaElements/simpla-richtext-behavior#28]
This shouldn't be a Simpla behavior, it's really just a general inline-editing richtext plugin. 

@bedeoverend thoughts? Can live either in simplaio repo or your personal one IMO.

**Comment by bedeoverend** [comment]
@seaneking agreed - happy to take this one on in personal repo if we're trying to trim down the Simpla repos, but not fussed if think we should keep it in Simpla, either way, should be a JS module for sure.

**Comment by madeleineostoja** [comment]
👍 

Not fussed where this goes, your call. Going to close this org come friday though, so has to be moved somewhere.

**Comment by bedeoverend** [comment]
Let's just move over to simplaio, can look at moving it to personal one later if need

**Comment by madeleineostoja** [comment]
👍

**Comment by madeleineostoja** [comment]
The other option is to make this a legit ES6 mixin, and chuck it under `simpla/mixins/richtext.js`. Not sure what, if anything, you could sugar in a mixin though. Bit pointless if you still have to instantiate with a method.
